_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,308 @@ nav_order: 50
+2. From your dashboard, select the visualization from which you want to create an alert. 
+3. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/} icon on the upper-right of the visualization and then select **Alerting** from the **Options** dropdown menu, as shown in the following image.
 
-1. Choose **Dashboard** from the OpenSearch Dashboards main menu.
-2. From the **Dashboards** window, select **Create** and then choose **Dashboard**.
-3. Select **Add an existing**, then select the appropriate alerting visualization from the **Add panels** list. The visualization is added to the dashboard.
-4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}). 
-5. From the **Options** menu, select **Add alerting monitor**.
-6. Input information for **Monitor details** and **Triggers**.
-7. Choose **Create monitor**. The monitor is added to the visualization.  
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-options-context-menu.png" alt="Selecting Alerting from Options dropdown menu within a visualization" width="700"/>
 
-An example of these steps is shown in the following screenshot.
+4. From the **Alerting** dropdown menu, select **Add alerting monitor**, as shown in the following image.
 
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/create-monitor-menu.png" alt="Create monitor interface" width="400" height="400">
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/add-monitor.png" alt="Adding an alerting monitor from within a visualization" width="700"/>
 
-## Associating monitors
+5. Enter the monitor details within the **Add alerting monitor** pop-up window and then save the new alerting monitor by selecting the **Create monitor** button, as shown in the following image. A new monitor is now created for the visualization.
 
-You can associate certain monitor types with a visualization using the Dashboard interface instead of the plugin page, giving you a single interface through which to add, view, and edit monitor data.
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/save-new-monitor.png" alt="Saving an alerting monitor from within a visualization" width="700"/>
 
-Continuing with the alerting visualization and dashboard created in the preceding section, associate an existing monitor with a visualization by following these steps: 
+### Associating monitors
 
-1. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}).
+You can associate certain monitor types with a visualization using **Dashboards** instead of the plugin page. This association links the monitor to automatically display alerts on the related visualization chart. 
```

Comment:
What's "plugin page"?

Reply:
```suggestion
You can associate certain monitor types with a visualization using **Dashboards** instead of through **OpenSearch Plugins Alerting**. This association links the monitor to automatically display alerts on the related visualization chart. 
```